.k8s/search-deployment.yml

```diff
@@ -0,0 +1,112 @@
+# Search Service deployment that creates a container for the app. It also mounts a volume with the Spring Boot
+# configuration file via a manually added ConfigMap called "search-config-map"
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+    name: search-deployment
+    labels:
+        app: search
+spec:
+    selector:
+        matchLabels:
+            app: search
+    template:
+        metadata:
+            labels:
+                app: search
+        spec:
+            containers:
+              - name: search
```

Comment:
❓ Is "docker.io" for dockerhub images?  If so, will these images ever be on dockerhub?

Reply:
No, we have a registry that we'll push them to just like we did for our other deployment

Reply:
I changed this to tag it to the correct registry.